man/rmd-fragments/authors-configuration.Rmd

```diff
@@ -0,0 +1,95 @@
+## Authors
+
+By default, pkgdown will display author information in three places:
+
+* the sidebar,
+* the left part side of the footer,
+* the author page.
+
+Both the sidebar and footer are customizable so you could move the authors 
+information or remove it entirely from them, see the sidebar and footer docs
+in respectively `?build_home` and `?build_footer`
+
+### Authors ORCID and bio
+
+Author ORCID identification numbers in the `DESCRIPTION` are linked using the ORCID logo:
+
+```
+Authors@R: c(
+    person("Hadley", "Wickham", , "hadley@rstudio.com", role = c("aut", "cre"),
+      comment = c(ORCID = "0000-0003-4757-117X")
+    ),
+    person("Jay", "Hesselberth", role = "aut",
+      comment = c(ORCID = "0000-0002-6299-179X")
+    )
+  )
+```
+
+If you want to add more details about authors or their involvement with the package,
+you can use the comment field,
+that will be rendered on the authors page.
+
+```
+Authors@R: c(
+    person("Hadley", "Wickham", , "hadley@rstudio.com", role = c("aut", "cre"),
+      comment = c(ORCID = "0000-0003-4757-117X", "Indenter-in-chief")
+    ),
+    person("Jay", "Hesselberth", role = "aut",
+      comment = c(ORCID = "0000-0002-6299-179X")
+    )
+  )
+```
+
+### YAML config - authors
+
+You can tweak a few things via the `authors` YAML field:
+
+* display of authors in the footer, sidebar and authors page,
+* which author roles are displayed in the sidebar and footer,
+* text before authors in the footer, 
+* text before and after authors on the authors page.
+
+
+You can modify the authors' display on the home page and author page 
+by adding a subsection for `authors`. 
+Each entry in `authors` should be named 
+with the author's name (matching `DESCRIPTION`)
+and can contain `href` and/or `html` fields:
+
+* If `href` is provided, the author's name will be linked to this URL.
+* If `html` is provided, it will be shown instead of the author's name.
+  This is particularly useful if you want to display the logo of a corporate
```

Comment:
Should we markdownify this too?

Reply:
Not sure, since it's actually easier to resize images with HTML :thinking:

Reply:
You can still use HTML

Reply:
I have just tried this on the pkgdown website and it adds space.
This is because `markdown_text2()` adds a `p`.
`commonmark::markdown_html("<img src='https://www.tidyverse.org/rstudio-logo.svg' alt='RStudio' height='24' />")` wouldn't.

``` r
img <- "<img src='https://www.tidyverse.org/rstudio-logo.svg' alt='RStudio' height='24' />"
pkgdown:::markdown_text2(img, list())
#> [1] "<p><img src=\"https://www.tidyverse.org/rstudio-logo.svg\" alt=\"RStudio\" height=\"24\"></p>"
commonmark::markdown_html(img)
#> [1] "<img src='https://www.tidyverse.org/rstudio-logo.svg' alt='RStudio' height='24' />\n"

text <- "[bla](bla)"
pkgdown:::markdown_text2(text, list())
#> [1] "<p><a href=\"bla\">bla</a></p>"
commonmark::markdown_html(text)
#> [1] "<p><a href=\"bla\">bla</a></p>\n"

text <- "bla\n\nbla\n\nbla"
pkgdown:::markdown_text2(text, list())
#> [1] "<p>bla</p><p>bla</p><p>bla</p>"
commonmark::markdown_html(text)
#> [1] "<p>bla</p>\n<p>bla</p>\n<p>bla</p>\n"
```

<sup>Created on 2021-02-23 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0.9001)</sup>

Reply:
Maybe a Pandoc option would help get a more similar result in the first case.

Reply:
Alternatively, maybe when there's only one p on the outside, `markdown_text2()` should remove it?

Reply:
I'll do this change